FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -124,6 +124,54 @@ The browser verifies it witnessed a Fetch request to the `seller`’s origin wit
 
 ## Device Orchestrated Multi-Seller Auctions
 
+Bidding and Auction Services auctions can also be used as component auctions. It
+is possible for multiple Bidding and Auction Services auctions to be used as
+component auctions for an on-device top-level auction.
+
+For efficiency, a top-level seller can construct encrypted requests for multiple
+component sellers in a single call to `navigator.getInterestGroupAdAuctionData()`:
+
+```javascript
+const result = await navigator.getInterestGroupAdAuctionData({
+  // ‘seller’ works the same as for runAdAuction.
+  'sellers': [
+    {
+      'seller': 'https://www.component-ssp1.example.com',
+      'coordinatorOrigin': 'https://publickeyservice.pa.gcp.privacysandboxservices.com/',
+    },
+    {
+      'seller': 'https://www.component-ssp2.example.com',
+      'coordinatorOrigin': 'https://publickeyservice.pa.gcp.privacysandboxservices.com/',
+  }],
+  // 'requestSize' the affects the size of the returned request (optional).
+  'requestSize': 51200,
```

Comment:
This setup requires a top-level seller driven one-size-fits-all requestSize across all component sellers.

Reply:
Yes, that's correct. Changing these other parameters has a higher performance impact as there is less that the browser can reuse.